microceph/ceph/run.go

```diff
@@ -3,3 +3,23 @@ package ceph
 func cephRun(args ...string) (string, error) {
 	return processExec.RunCommand("ceph", args...)
 }
+
```

Comment:
I am afraid this file is not the correct place for these helpers (except `radosRun` to go.